src/npc.cpp

```diff
@@ -617,7 +618,7 @@ int NpcScriptInterface::luaActionSay(lua_State* L)
 int NpcScriptInterface::luaActionMove(lua_State* L)
 {
 	// selfMove(direction)
-	Npc* npc = tfs::lua::getScriptEnv()->getNpc();
+	auto npc = tfs::lua::getScriptEnv()->getNpc();
```

Comment:
Could this be `const auto`
```suggestion
	const auto npc = tfs::lua::getScriptEnv()->getNpc();
```